docs/Change_Log.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+#### February XX, 2021
```

Comment:
should add a `##` title above this for this one and the Jan 22 one to be consistent with all the others

Reply:
Yeah I've realized we're actually missing as few headings in the changelog. Will do a pass